src/deploy/standalone/noobaa_syslog.conf

```diff
@@ -4,6 +4,16 @@ $umask 0000
 $FileCreateMode 0644
 $EscapeControlCharactersOnReceive off
 
+$imjournalRatelimitInterval 0
+$imjournalRatelimitBurst 0
+
+# start log rotation via outchannel
+# outchannel definition, 100M limit, If updating this value update logrotate also.
+$outchannel log_rotation,/var/log/noobaa.log,104857600,bash /etc/logrotate.d/noobaa/logrotate_noobaa.sh
+#  activate the channel and log everything to it
+*.* :omfile:$log_rotation
```

Comment:
this seems to apply the same outchannel to all interfaces - isn't that messing up the other 2 interfaces we have? should we define separate outchannels for them? or maybe we can just set this channel only to local0.* ?

Reply:
Ya, will set only for the `local0.*`